### DISCUSSION TITLE
Time before it goes back to home page ?

### DISCUSSION THREAD

**Discussion by adri123** [Blackymas/NSPanel_HA_Blueprint#997] [Q&A] [answered]
Hi folks, and thanks for your amazing work.
I'm still wondering after some searches on the github pages if there is a way to set how long it takes to go back to main screen ddddwhen navigating in the pages ?
Maybe I miss the information, then I'm sorry for the dumb question.
Have a good day.

**Comment by edwardtfn** [comment, marked as answer]
Go to the device page in Home Assistant (Settings > Devices and Services > ESPHome) and set the page timeout under configuration:
![IMG_6401](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/55d681d5-0256-4b62-8485-79cd36dd679c)

**Reply by adri123** [reply]
Great ! Thank you,
